src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -793,10 +794,6 @@ module FrameworkDetection =
                 | "sl4-wp75" | "sl4-wp7.5" -> Some (WindowsPhone WindowsPhoneVersion.V7_5)
                 | MatchTfms ["wp";"wpv"] (Bind WindowsPhoneVersion.TryParse) fm -> Some (WindowsPhone fm)
                 | MatchTfms ["wpa";"wpav";"wpapp"] (Bind WindowsPhoneAppVersion.TryParse) fm -> Some (WindowsPhoneApp fm)
-                | MatchTfm "dnx" (allowVersions ["";"4.5.1"]) () -> Some(DNX FrameworkVersion.V4_5_1)
-                | MatchTfms ["dnxcore";"netplatform";"netcore";"aspnetcore";"aspnet";"dotnet"] (Bind (allowVersions ["";"5"]))
-                    () -> Some(DNXCore FrameworkVersion.V5_0)
-                | v when v.StartsWith "dotnet" -> Some(DNXCore FrameworkVersion.V5_0)
```

Comment:
so we drop support for those monikers and hope no-one uses them? (fine by me just asking)
Do we tell those people to stay on previous paket versions?

Reply:
those things do not exist. they are completely abandoned. I wanted to drop them long time ago.

Reply:
NuGet apparently still parses them: http://nugettoolsdev.azurewebsites.net/4.5.0/parse-framework?framework=dnxcore
Yes, we can remove them and see what happens but I wouldn't be surprised if this at least produces warnings for some packages still having those things. I just wanted to check if we have a strategy if this is indeed the case or if we need one? For example we could tell them to update their packages and tell the package author to fix its package or revert the change.